### PR TITLE
Dev constantin matei intro

### PR DESCRIPTION
**Pull request by Constantin-Matei**
Started My CV v2 Junior :)

### CODE REVIEW

**Review by marianzburlea — COMMENTED** [review]
bravo

**Review by MariaTeo — APPROVED** [review]
Nice 💯

**Review by biggicode — APPROVED** [review]
Well done!

**Review by marianzburlea — APPROVED** [review]
💯

File: src/_component/about-me/about-me.scss [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  margin: 20px auto 0;
+  color: white;
+  border: 0;
+  border-radius: 20px;
+  padding: 0 24px;
+  display: flex;
+  align-items: center;
+}
+
+  .buton__pastiluta-albastra {
```
Comment (marianzburlea) [inline src/_component/about-me/about-me.scss:121]: bad indentation :)

**Review by marianzburlea — APPROVED** [review]
💯 💯 💯 💯 💯 💯

**Pull request merged**